kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/CertificateGeneratorNetty.java

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
This class will be removed when a new release of kroxylicious-junit5-extension is created with the changes on the CertificateGenerator.